modules/builds-strategy-pipeline-mapping-buildconfig-jenkins.adoc

```diff
@@ -33,3 +33,11 @@ with `oc start-build -e` takes precedence.
 * If you start the Jenkins job with the Jenkins console, then you can control the
 setting of the parameters with the Jenkins console as part of starting a build
 for the job.
+
+[NOTE]
+====
+As noted above, while it is not required, It is still recommended that you specify all possible
+environment variables / job parameters in the `BuildConfig`.  Doing so reduces
```

Comment:
s/variables / job/variables and job

Reply:
actually that didn't read great either ... updated in a different way, see what you think